code/modules/reagents/reagent_containers/food.dm

```diff
@@ -22,6 +22,7 @@
 	center_of_mass = @'{"x":16,"y":16}'
 	w_class = ITEM_SIZE_SMALL
 	abstract_type = /obj/item/chems/food
+	needs_attack_dexterity = DEXTERITY_NONE
```

Comment:
I don't like this. We confirmed drakes can eat foods fine by using it in-hand, which I thought was the intended way for them to do it? This only affects clicking themselves (and other things) with food, and will undoubtedly have other side effects.

Reply:
it was misreported, we confirmed it on discord. using it inhand worked, clicking yourself is what didn't and that's intended afaik

Reply:
in that case yeah, this is fine